compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__recursive_var.snap

```diff
@@ -14,5 +14,4 @@ error: Recursive type
 
 I don't know how to work out what type this value has. It seems to be
 defined in terms of itself.
-
```

Comment:
I switched to using the `Diagnostic`'s `hint` field which got rid of the empty line. Now it's consistent with all the other hints we show in error messages